lua/plenary/path.lua

```diff
@@ -474,20 +474,77 @@ function Path:rename(opts)
   return status
 end
 
+--- Copy files or folders.
+---@param opts table: options to pass to toggling registered actions
+---@field destination string|Path: target file path to copy to
+---@field recursive bool: whether to copy folders recursively (default: true)
+---@field override bool: whether to override files (default: true)
+---@field interactive bool: confirm if copy would override; precedes `override` (default: false)
```

Comment:
does it make sense to have interactive default true ? Does file_browser disable it ? Because if `vim.select` is set to telescope it will close it